sympy/polys/polytools.py

```diff
@@ -4164,6 +4163,56 @@ def galois_group(f, by_name=False, max_tries=30, randomize=False):
         G = name if by_name else name.get_perm_group()
         return G, alt
 
+    def routh_hurwitz_stability(f):
+        """
+        Return the conditions for the polynomial to be stable (all roots with
+        negative real part).
+        The conditions, in general, are represented by a list of multivariate
+        polynomials which must be positive to ensure stability.
+
+        Note
+        =====
+        This method assumes that the leading coefficient is non-zero.
```

Comment:
This docstring is a bit long and not formatted in the standard way. It should begin with a single line summary. Headers like "Note" should have a blank line underneath. It is not necessary to include more than one example.